pkg/types/types.go

```diff
@@ -58,6 +58,12 @@ func NewNodeIDFromInt(id int) NodeID {
 	return NodeID(strconv.Itoa(id))
 }
 
+func (id NodeID) NodeIDToUint64() uint64 {
+	idUint, _ := strconv.ParseUint(string(id), 10, 64)
+	// TODO Handle error
+	return idUint
+}
+
```

Comment:
After addressing the previous comments, this function should not be necessary. If it still is, it indicates a problem somewhere.